tests/unit/drivers/structure_run/test_local_structure_run_driver.py

```diff
@@ -23,7 +23,7 @@ def test_run_with_env(self, mock_config):
         mock_config.drivers_config.prompt_driver = MockPromptDriver(mock_output=lambda _: os.environ["KEY"])
         agent = Agent()
         driver = LocalStructureRunDriver(structure_factory_fn=lambda: agent, env={"KEY": "value"})
-        task = StructureRunTask(driver=driver)
+        task = StructureRunTask(structure_run_driver=driver)
```

Comment:
the field needs to be updated to `structure_run_driver` i think

Reply:
Oh shoot I meant to undo that particular change.